Feature/flat ged destination

LrxGaelle opened the pull request:
https://platform-dev-flat-ged-destination.bimdata.io

## Description
<!--- Describe your changes in detail -->

## Types of changes
<!--- Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
<!--- Put an `x` in all the boxes that apply: -->
- [ ] My code follows the code style of this project.
- [ ] I have tested my code.
- [x] I want to run the tests for the commits of this PR

Amoki commented on src/components/cloud-list/CardCloudList.vue:
```diff
@@ -134,7 +270,7 @@ export default {
       this.displayRename = false
     },
-    accessCloud () {
+    accessCloud (info) {
```
?

Amoki commented on src/components/pages/PageCloudList.vue:
```diff
@@ -119,6 +119,9 @@ export default {
     filteredClouds () {
       let filteredClouds = this.clouds.filter(cloud => {
+        if (!cloud.name) {
+          return []
```
```suggestion
          return false
```

LrxGaelle committed:
create getAscendants function

LrxGaelle committed:
create FilePathCell view for FilesTable component

LrxGaelle committed:
add location path for files in `all files` tab view

LrxGaelle committed:
whitespace linter

LrxGaelle committed:
delete unused getAscendants function

LrxGaelle committed:
delete comment

LrxGaelle committed:
@bimdata/design-system

LrxGaelle committed:
create FilePathCell component :
- create a 'breadcrumb' for parent folders of file on click
- create style for component

LrxGaelle committed:
add Textbox for last folder location

LrxGaelle committed:
add click to return to folder view on `root folder`

LrxGaelle committed:
add hover & translations

LrxGaelle committed:
add location translation

LrxGaelle committed:
fix full path location submenu & use translations

LrxGaelle committed:
fix padding

LrxGaelle committed:
create VisasView component & create myVisas computed

LrxGaelle committed:
jump to `folders` view on location folder click

LrxGaelle committed:
Revert "create VisasView component & create myVisas computed"

This reverts commit 89e6d65cb0d1dd0f251a6e5c345d68442538ef03.

LrxGaelle committed:
fix search allFiles

LrxGaelle committed:
fix fileNameCell width

LrxGaelle committed:
Added long name management and management of more than 4 folders

LrxGaelle committed:
fix FileNameCell width

LrxGaelle committed:
fix created_by & lastupdate columns width

LrxGaelle committed:
fix responsive for all files view

LrxGaelle committed:
add fade transition

NicolasRichel commented on src/components/specific/files/files-table/file-name-cell/FileNameCell.scss:
```diff
@@ -1,4 +1,5 @@
 .file-name-cell {
+  // max-width: calc(773.5px - (var(--spacing-unit) * 2));
```
Commented code here. ;)

LrxGaelle committed:
delete comment code

Bimdata-io commented:
:tada: This PR is included in version 1.0.0 :tada:

The release is available on [GitHub release](https://github.com/bimdata/platform/releases/tag/v1.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
